mentoring/public/js/mentoring_assessment_view.js

```diff
@@ -127,49 +131,64 @@ function MentoringAssessmentView(runtime, element, mentoring) {
         return $(element).find('div[name="' + name + '"]');
     }
 
-    function jumpToName (event) {
+    function jumpToName(event) {
         // Used only during extended feedback. Assumes completion and attempts exhausted.
         event.preventDefault();
-        var options = {
-            onChange: onChange
-        };
 
         var target = getByName($(event.target).data('name'));
-        active_child = $.inArray(target[0], mentoring.children_dom);
+        reviewDisplayChild($.inArray(target[0], mentoring.children_dom), {});
+    }
+
+    function reviewDisplayChild(child_index, options) {
+        active_child = child_index;
         cleanAll();
         mentoring.displayChild(active_child, options);
         mentoring.publish_event({
             event_type: 'xblock.mentoring.assessment.review',
-            exercise_id: $(target).attr('name')
+            exercise_id: $(mentoring.children_dom[active_child]).attr('name')
         });
         post_display(true);
         get_results();
     }
 
+    function reviewNextChild(event) {
+        nextDOM.attr('disabled', 'disabled')
+        nextDOM.hide()
+        findNextChild({})
```

Comment:
nit: Instead of having to pass an empty object to `findNextChild` it would be nicer if the argument optional.